lua/elixir/projectionist/init.lua

```diff
@@ -32,14 +32,32 @@ local config = {
       },
     },
     ["test/**/controllers/*_controller_test.exs"] = {
-      alternate = "lib/{dirname}/controllers/{basename}_controller.ex",
       type = "test",
+      alternate = "lib/{dirname}/controllers/{basename}_controller.ex",
       template = {
         "defmodule {dirname|camelcase|capitalize}.{basename|camelcase|capitalize}ControllerTest do",
         "  use {dirname|camelcase|capitalize}.ConnCase, async: true",
         "end",
       },
     },
+    ["lib/**/live/*_live.ex"] = {
+      type = "liveview",
+      alternate = "test/{dirname}/live/{basename}_live.exs",
+      template = {
+        "defmodule {dirname|camelcase|capitalize}.{basename|camelcase|capitalize}Live do",
+        "  use {dirname|camelcase|capitalize}, :live_view",
+        "end",
+      },
+    },
+    ["test/**/live/*_live.exs"] = {
+      type = "test",
+      alternate = "lib/{dirname}/live/{basename}_live.ex",
+      template = {
+        "defmodule {dirname|camelcase|capitalize}.{basename|camelcase|capitalize}LiveTest do",
+        "  use {dirname|camelcase|capitalize}.ConnCase, async: true",
```

Comment:
Do LiveView tests use ConnCase?

Reply:
Yep! Technically it looks like a generated test using the newest 1.7rc2 has a consistent:

```elixir
defmodule TestWeb.DocumentLiveTest do
  use TestWeb.ConnCase

  import Phoenix.LiveViewTest
```
it has ConnCase, though it doesn't set async to true and it has an additional LiveViewTest import as well.